mkdocs.yml

```diff
@@ -251,6 +251,7 @@ nav:
           - gdb-oneapi: aurora/debugging/gdb-oneapi.md
           - DDT: aurora/debugging/ddt-aurora.md
           - gdb4hpc: aurora/debugging/gdb4hpc.md
+          - codee: aurora/debugging/codee.md
```

Comment:
I see it capitalized on their website: https://www.codee.com/about-us/
and on NERSC docs?

Reply:
Good point, I'll update it!